package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
```

Comment:
Can this be a dev dependency instead of dependency?

Reply:
moved to dev dep